indica/operators/mean_charge.py

```diff
@@ -73,21 +73,21 @@ def __call__(self, FracAbundObj: DataArray, element: str):  # type: ignore
         )
         input_check("element", element, str)
 
-        element_atomic_number_tmp = [
-            k for k, v in ELEMENTS_BY_ATOMIC_NUMBER.items() if v == element
-        ]
         try:
-            assert len(element_atomic_number_tmp) == 1
+            assert element in ELEMENTS.keys()
         except AssertionError:
             raise ValueError(
                 f"Please input a single valid element from list:\
-                {list(ELEMENTS_BY_ATOMIC_NUMBER.values())}"
+                {list(ELEMENTS.keys())}"
             )
 
-        element_atomic_number = element_atomic_number_tmp[0]
+        element_atomic_number = ELEMENTS[element][0]
 
         ionisation_charges = np.linspace(
```

Comment:
This could be simplified to just `np.arange(0, element_atomic_number + 1)` looking at the fact it's setting step to `element_atomic_number + 1`. Would also remove the need for `dtype=int` (although it doesn't hurt to keep that just to enforce it)

Reply:
You're right, it's simpler but `mypy` still throws an error even with `dtype=int` being specified. Either way I'll change it to use `np.arange`